ci/azure-pipelines-release.yml

```diff
@@ -32,7 +30,7 @@ stages:
               TARGET: windows-amd64
         steps:
           - checkout: self
-            path: 'go/src/github.com/hyperledger/fabric'
+            path: 'fabric'
```

Comment:
I'm a little concerned about this change since I don't know the full contents of the golang docker image. In particular whether or not `$(go env GOPATH)/bin` is on the path.

Then again, since this is just building, it may not matter.

Reply:
This should be alright, but I'll test it out of band against my own repo